openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -1,24 +1,31 @@
 """
 This module defines SafeSessionMiddleware that makes use of a
 SafeCookieData that cryptographically binds the user to the session id
-in the cookie.
+in the cookie (but does not reveal the user ID to the holder).
```

Comment:
Hmm, cancel this bit -- an attacker in possession of a session cookie and nothing else could still discover the user ID by brute-forcing the hash (iterating through all 7-digits integers, for example).

Reply:
(which is OK, but it would be cool if the user ID weren't discoverable)